VogtFactory/RentalCar.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtFactory
+{
+    class RentalCar
+    {
+        public string Name;
+        public int NumberOfPeopleThatCanFit;
+
+        RentalCarFactory rcf = new RentalCarFactory();
+
+        public virtual void DisplayCarInfo()
+        {
+            Console.WriteLine($"{Name} {NumberOfPeopleThatCanFit}");
```

Comment:
1. Update this to:
`Console.WriteLine($"{Name} car can fit {NumberOfPeopleThatCanFit}");`
2. Remove the `virtual`
3. Remove the `DisplayCarInfo` from all of the other classes EXCEPT for this one

Let this function work for ALL rental cars via polymorphism!